src/page/home/home.tsx

```diff
@@ -27,6 +29,9 @@ function Home() {
     onDragEnd,
   } = useHome();
 
+const [editingTaskId, setEditingTaskId] =  useState<string |  null>(null);
```

Comment:
For pages, we’ve separated the logic from the UI using a custom hooks. For this, we use `useHome()`, which handles all the necessary state, references, and functions. This keeps the component clean and focuses solely on rendering the UI while keeping business logic in the hook for better reusability and maintainability.